pkgs/development/ocaml-modules/janestreet/buildOcamlJane.nix

```diff
@@ -1,14 +1,14 @@
 { buildOcaml, opaline, js_build_tools, ocaml_oasis, fetchurl } :
 
-{ name, version ? "113.33.03", buildInputs ? [],
+{ pname, version ? "113.33.03", buildInputs ? [],
   hash ? "",
   minimumSupportedOcamlVersion ? "4.02", ...
 }@args:
 
 buildOcaml (args // {
-  inherit name version minimumSupportedOcamlVersion;
+  inherit pname version minimumSupportedOcamlVersion;
   src = fetchurl {
```

Comment:
This should use fetchFromGitHub, right?

Reply:
i suggest this will be a separate PR after merging of #144128.

Reply:
works for me